src/components/views/elements/Pill.js

```diff
@@ -229,6 +246,20 @@ const Pill = React.createClass({
                 }
             }
                 break;
+            case Pill.TYPE_GROUP_MENTION: {
+                if (this.state.group) {
+                    const {avatarUrl, groupId, name} = this.state.group;
+                    const cli = MatrixClientPeg.get();
+
+                    linkText = groupId;
+                    if (this.props.shouldShowPillAvatar) {
+                        avatar = <BaseAvatar name={name || groupId} width={16} height={16}
+                                             url={avatarUrl ? cli.mxcUrlToHttp(avatarUrl, 16, 16) : null} />;
+                    }
+                    pillClass = 'mx_RoomPill' || 'mx_GroupPill';
```

Comment:
Not sure what this is supposed to be doing. Should this not always be `mx_GroupPill`?

Reply:
Crap, this was a temporary measure before I applied the CSS to `mx_GroupPill` sorry about this.